tests/test_storage_mount.py

```
@@ -0,0 +1,617 @@
+    command = _hook_command()
+    result = _run_hook(command, bypass, tmp_path, containers_running=False)
+    assert "deny" in result.stdout, f"{bypass!r} was not blocked"
+
+
+def test_bypass_refusal_explains_the_supported_path():
+    """A refusal that does not say what to do instead invites working around
+    it, which is how --no-verify got used in the first place."""
+    payload = json.dumps(
+        {"tool_name": "Bash", "tool_input": {"command": "git commit --no-verify -m x"}}
+    )
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)